mint/run/core/aws-sdk-go/quick-tests.go

```diff
@@ -761,6 +847,7 @@ func main() {
 	testSelectObject(s3Client)
 	testCreateBucketError(s3Client)
 	testListMultipartUploads(s3Client)
+	testSSECopyObject(s3Client)
```

Comment:
`SSEC` tests require `https`. This test should only be run when the connection is `https`